src/FormExtension/Actions/EnqueueFormBuilderScripts.php

```diff
@@ -43,27 +43,49 @@ public function __construct(\ActiveCampaign $activecampaign)
      */
     public function __invoke()
     {
-        $tags = json_decode($this->activecampaign->api( 'tags/list', [ 'ids' => 'all' ] ));
-
         wp_enqueue_script('givewp-form-extension-activecampaign', $this->scriptSrc, $this->scriptAsset['dependencies']);
         wp_localize_script('givewp-form-extension-activecampaign', 'GiveActiveCampaign', [
-            'lists' => $this->getLists(),
-            'tags' => array_map(function ($tag) {
-                return ['value' => $tag->name, 'label' => $tag->name];
-            }, $tags),
+            'requiresSetup' => !$this->activecampaign->credentials_test(),
+            'settingsUrl'   =>  '/wp-admin/edit.php?post_type=give_forms&page=give-settings&tab=activecampaign',
```

Comment:
A relative URL does not need to be localized, since it is a known string.

I would suggest wrapping the relative URL in [admin_url()](https://developer.wordpress.org/reference/functions/admin_url/) for a fully qualified URL (which is unique to the site. This will also handle `http`/`https`.

```suggestion
            'settingsUrl'   =>  admin_url('edit.php?post_type=give_forms&page=give-settings&tab=activecampaign'),
```

Reply:
Ok that make's sense. I'll keep this in mind & make sure to update the other block requirements as well. Thanks. Resolved: 139d0a1a4a312c31e61da74f6916e38708aedd9c